process/txcost/converters.go

```diff
@@ -0,0 +1,59 @@
+package txcost
+
+import (
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-proxy-go/data"
+)
+
+const atSep = "@"
+
+func (tcp *transactionCostProcessor) computeSenderAndReceiverShardID(sender, receiver string) (uint32, uint32, error) {
```

Comment:
duplicated code in function
create a new function `getAddressShardID(address string) (uint32, error)` and use it in one of these ways:
1. keep this `computeSenderAndReceiverShardID` function and call the new function
2. call the `getAddressShardID` twice where you need the shard ids